index.src.html

```diff
@@ -4433,27 +4553,236 @@
     {{VideoFrame/codedWidth}} by 2, rounded up to the nearest integer. Samples
     are arranged starting at the top left of the image.
 
-    The visible rectangle offset ({{VideoFrame/visibleRect}}.{{DOMRectInit/x}})
+    The visible rectangle horizontal offset
+    ({{VideoFrame/visibleRect}}.{{DOMRectInit/x}})
     <em class="rfc2119">MUST</em> be even.
   </dd>
-  <dt><dfn enum-value for=VideoPixelFormat>I444</dfn></dt>
+  <dt><dfn enum-value for=VideoPixelFormat>I422P10</dfn></dt>
   <dd>
+    This format is composed of three distinct planes, one plane of Luma and two
+    planes of Chroma, denoted Y, U and V, and present in this order.
+
+    The U an V planes are [=sub-sampled=] horizontally by a [=factor=] of 2
+    compared to the Y plane, and not [=sub-sampled=] vertically.
+
+    Each sample in this format is 10 bits, encoded as a 16-bit integer in
+    little-endian byte order.
```

Comment:
I wonder if we should say something about those 6 bits (or 4, depending on the depth) that aren't used.

Are they to be ignored, are they always set to 0? In terms of developer ergonomics, if I read back the values to an `ArrayBuffer` to perform some computations, should I always mask the unused bits or can I simply read the value with a `DataView` as uint16_t.  In terms of implementation, is that a fair assumption we can make, or do some decoders leave the bits alone, and so this could be a source of fingerprinting?

We should also specify (probably globally, since it applies to all formats) that the sample values are unsigned integer, I tried to look for that but didn't find it for the video side of things.

Reply:
Looks good to me too.